pkg/rkenodeconfigserver/nodeserver.go

```diff
@@ -321,3 +322,22 @@ func appendTaintsToKubeletArgs(processes map[string]v3.Process, nodeConfigTaints
 	}
 	return processes
 }
+
+func (n *RKENodeConfigServer) getServiceOptions(k8sVersion string) map[string]interface{} {
+	data := map[string]interface{}{}
+	svcOptions, err := kd.GetRKEK8sServiceOptions(k8sVersion, n.serviceOptionsLister, n.serviceOptions)
+	if err != nil {
+		logrus.Errorf("getK8sServiceOptions: k8sVersion %s [%v]", k8sVersion, err)
+	}
```

Comment:
Why dont we return err here? I see that kd.GetRKEK8sServiceOptions returns error and also the caller can return errors.